.pre-commit-config.yaml

```diff
@@ -49,3 +49,9 @@ repos:
     hooks:
     - id: ruff
       args: [--fix]
+
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+    - id: check-added-large-files
+      args: [--maxkb=5000]
```

Comment:
Whats the lowest this can go right now?

Reply:
Maybe like 3-4 MB? Currently also wanted to keep a nice round number.